src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1285,7 +1285,7 @@
     <value>'Stream' has a 'ReadAsync' overload that takes a 'Memory&lt;Byte&gt;' as the first argument, and a 'WriteAsync' overload that takes a 'ReadOnlyMemory&lt;Byte&gt;' as the first argument. Prefer calling the memory based overloads, which are more efficient.</value>
   </data>
   <data name="PreferStreamAsyncMemoryOverloadsMessage" xml:space="preserve">
-    <value>Change the '{0}' method call to the overload that receives a '{1}'.</value>
+    <value>Change the '{0}' method call to the overload '{1}'.</value>
```

Comment:
Not relate to your change but it seems like a good opportunity to fix this. Message should not end with a period.

Reply:
@Evangelink I've removed the period.